Composer/packages/extensions/visual-designer/src/constants/ScreenReaderMessage.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export enum ScreenReaderMessage {
+  EventFocused = 'Event focused',
+  ActionFocused = 'Action focused',
+  DialogOpened = 'Dialog opened',
+  ActionDeleted = 'Action deleted',
+  ActionsDeleted = 'Actions deleted',
+  ActionCreated = 'Action created',
+  ActionsCreated = 'Actions created',
+  EventCreated = 'Event created',
+  ActionsCopied = 'Actions copied',
+  ActionsCut = 'Actions cut',
+  ActionsMoved = 'Actions moved',
+  ActionUndo = 'Action undo',
+  ActionRedo = 'Action redo',
+}
```

Comment:
I think these could just be "undo" and "redo".

Reply:
Sure.